### DISCUSSION TITLE
How to skip rate limit check when Redis is down?

### DISCUSSION THREAD

**Discussion by liron-tepper** [MarcGiffing/bucket4j-spring-boot-starter#122] [Q&A] [answered]
Hello @MarcGiffing, @vladimir-bukhtoyarov,

When Redis is down, I am getting a RedisConnectionFailureException.
I would like to check if there is an option in such case to totally disable the rate limit check (and allow the API to be called).

Thanks a lot,
Liron

**Comment by MarcGiffing** [comment, marked as answer]
You may add a Skip Condition to your filter:

bucket4j.filters[0].rate-limits[0].skip-condition=@redisStatusCheck.isDown()
or
bucket4j.filters[0].rate-limits[0].execute-condition=@redisStatusCheck.isUp()

You have to provide a Bean with the name redisStatus with implements the example methods.
